deps.ts

```diff
@@ -1,22 +1,34 @@
 import 'https://esm.sh/@abraham/reflection@0.12.0';
+import { unmanaged as _unmanaged } from 'https://esm.sh/inversify@6.0.1';
 
 export { green, red } from 'https://deno.land/std@0.132.0/fmt/colors.ts';
 
 export * as semver from 'https://deno.land/x/semver@v1.4.0/mod.ts';
 
 export { type Args, parse } from 'https://deno.land/std@0.132.0/flags/mod.ts';
 
-export {
-  container,
-  type DependencyContainer,
-  type FactoryFunction,
-  inject,
-  injectable,
-  injectAll,
-  predicateAwareClassFactory,
-  registry,
-  singleton,
-} from 'https://esm.sh/tsyringe@4.7.0';
+// export {
+//   container,
+//   type DependencyContainer,
+//   type FactoryFunction,
+//   inject,
+//   injectable,
+//   injectAll,
+//   predicateAwareClassFactory,
+//   registry,
+//   singleton,
+// } from 'https://esm.sh/tsyringe@4.8.0';
+
+export * as inversify from 'https://esm.sh/inversify@6.0.1';
+// Next line is a temporary shim.
+
+export const unmanaged = _unmanaged as () => (
```

Comment:
This is a work around for some minor weirdness inversify is having with the unmanaged decorator in 6.0.1. Should add some documentation to note this.